phpstan.neon

```diff
@@ -3,16 +3,35 @@ parameters:
         - tests/Fixtures/app/AppKernel.php
     excludes_analyse:
         - tests/Fixtures/app/cache
+        # The Symfony Configuration API isn't good enough to be analysed
+        - src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php
     ignoreErrors:
-        - '#Call to an undefined method Symfony\\Component\\Routing\\Exception\\ExceptionInterface::getCode\(\)#'
-        - '#Call to an undefined method Prophecy\\Prophecy\\ObjectProphecy::[a-zA-Z0-9_]+\(\)#'
-        - '#Access to an undefined property Prophecy\\Prophecy\\ObjectProphecy::\$[a-zA-Z0-9_]+#'
-        - '#Call to an undefined method PHPUnit\\Framework\\MockObject\\MockObject::[a-zA-Z0-9_]+\(\)#'
-        - '#Call to an undefined method Doctrine\\Common\\Persistence\\Mapping\\ClassMetadata::getAssociationMappings\(\)#'
+        # Real problems, hard to fix
+        - '#Parameter \#2 \$dqlPart of method Doctrine\\ORM\\QueryBuilder::add\(\) expects array\|object, string given\.#'
 
         # False positives
+        - '#Access to an undefined property Prophecy\\Prophecy\\ObjectProphecy::\$[a-zA-Z0-9_]+#'
+        - '#Access to an undefined property object::\$isIdentifierComposite.#'
```

Comment:
If you add property_exists, it's recognized.

Reply:
Yes but as it’s in tests/mocks, I’m
not sure it is worth it. Maybe the Prophecy extension will fix this, but it’s not compatible with 0.10 yet.